newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -146,6 +146,9 @@ export default class PixiResourcesLoader {
     return loadedTextures[resourceName];
   }
 
+  //any better way to access this from JSExtensions.js?
+  static ResourcesLoader = ResourcesLoader;
```

Comment:
I still need to remove this, but when I do the ide crashes

Reply:
What's the error? you're maybe are referencing `ResourcesLoader` in a wrong way?

Reply:
I get TypeError: Cannot read property 'getResourceJsonData' of undefined

Reply:
Include the full strak trace when you report an error, otherwise it's hard to understand what is failing.

Reply:
I will give this another try tonight and put the trace here :)

Reply:
```
TypeError: Cannot read property 'getResourceJsonData' of undefined
RenderedTileMapInstance.updateTileMap
D:\DEV\GD\GDevelop\newIDE\app\resources\GDJS\Runtime\Extensions\TileMap\JsExtension.js:459:49
new RenderedTileMapInstance
D:\DEV\GD\GDevelop\newIDE\app\resources\GDJS\Runtime\Extensions\TileMap\JsExtension.js:417:12
createNewInstanceRenderer
D:/DEV/GD/GDevelop/newIDE/app/src/ObjectsRendering/ObjectsRenderingService.js:62
```

Reply:
I moved  getResourceJsonData to PixiResourceLoader to fix this :)

Reply:
Excellent, I was just writing a message advising this :)